views/partials/mlb_player_card.hbs

```diff
@@ -0,0 +1,37 @@
+<div>Bats: <span>Throws: </span> </div>
+<table id="stats-table" class="player-card-table">
+  <tr>
+    <th style="background-color: {{primary_hex}}">AVG</th>
```

Comment:
this is just temporary. not even getting hex for baseball yet. just copied the card over.